### PR TITLE
Totems HUD fixes

### PR DESCRIPTION
**Pull request by Ozynt**

### CODE REVIEW

**Comment by 3arthqu4ke** [comment]
How exactly does this help? Maybe you could just do it the way we do it for armor?

**Comment by Ozynt** [comment]
Sure, I will redo and just leave the settings name fix.

**Comment by 3arthqu4ke** [comment]
Can you do Totems-Y-Offset instead? Spaces in setting names are annoying since they need to get escaped when typing a command.

**Comment by Ozynt** [comment]
I added enableDepth at the end since it caused some wierd issue in the inventory for me.

File: src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java [hunk 1]
```diff
@@ -284,15 +284,15 @@ protected void renderModules() {
             int totems = InventoryUtil.getCount(Items.TOTEM_OF_UNDYING);
 
             if (totems > 0) {
-                int x = width / 2 - 7;
-                int y = height - (totemOffset.getValue()) - getArmorY();
+                int x = width / 2 - 8;
```
Comment (3arthqu4ke) [inline src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:287]: XOffsetSetting?

**Review by 3arthqu4ke — CHANGES_REQUESTED** [review]

File: src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java (hunk 1 above)
Comment (Ozynt) [reply src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:287]: Sure but would be pretty cursed

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:287]: ¯\_(ツ)_/¯

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:287]: No just for small adjustments. I felt like int x = width / 2 - 7; was fine, but if you dont make a setting for small offsets.

Comment (3arthqu4ke) [reply src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:287]: You can do big offsets too idc

Comment (Ozynt) [reply src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java:287]: Ok I will add a small one in a few hours.

**Review by 3arthqu4ke — APPROVED** [review]

**Pull request merged**